core/src/costs.rs

```diff
@@ -79,6 +79,14 @@ impl<T> Default for CostOf<T> {
 #[derive(Debug, Default, Clone, Copy, derive_more::From, derive_more::Into)]
 pub struct CallsAmount(u32);
 
+impl CostOf<CallsAmount> {
+    /// Calculate (saturating add) cost for `per_byte` amount of `BytesAmount` (saturating mul).
+    pub fn cost_for_with_bytes(&self, per_byte: CostOf<BytesAmount>, amount: BytesAmount) -> u64 {
```

Comment:
Just wondering, wasn't it supposed to be cost_of_with_bytes?

Reply:
:man_shrugging: